{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -50,24 +50,33 @@ docs-clean:
 docs-open:
 	$(OPEN) docs/_build/index.html
 
-.PHONY: conda-update
-## Update the conda-environment based on changes to `environment.yaml`
-conda-update:
-	conda env update -n ${REPO_NAME} -f environment.yaml
-	$(MAKE) -s pip-install
-	direnv reload
+.PHONY: python-env-create
+## Create the virtual environment
+python-env-create:
+    @echo "Creating ${ENVIRONMENT_TYPE} environment..."
+    @${ENV_CREATE}
+
+.PHONY: python-env-remove
+## Remove the virtual environment
+python-env-remove:
+    @echo "Removing ${ENVIRONMENT_TYPE} environment..."
+    -@${ENV_DEACTIVATE} 2>/dev/null || true
+    -@${ENV_REMOVE} 2>/dev/null || true
+    @rm -f .cookiecutter/state/python-env-create*
+    @direnv reload
+
+.PHONY: python-env-update
```

Comment:
This target doesn't ever seem to be called? Nor the `ENV_UPDATE` methods elsewhere outside this target. I guess just there in case users ever wanted to run `make python-env-update`? If so, maybe we should add to the documentation somewhere.

Reply:
Exploring the cookiecutter, seems there is documentation in `STRUCTURE.md` so makes sense to have the update targets I think! Just update that documentation, as some of the other targets documented there have chagned as well (currently they are all conda specific).

Reply:
Jesus never mind. That documentation is extensive, so likely needs a full overhaul when all changes are agreed and merged.

Reply:
Yeah for now was just going for similar commands to the original make files, but yes will have to sort doc I will drop a TODO for that.